src/packtPublishingFreeEbook.py

```diff
@@ -118,6 +126,7 @@ def __init__(self, currentSession):
         self.session = currentSession.getCurrentHttpSession()
         self.accountData = currentSession.getCurrentConfig()
         self.bookTitle = ""
+        self.pretty_book_title = None
```

Comment:
I think, there is no need to create another 'title' variable which will be only used for sending status email